src/domain/enrolment/enrolmentForm/EnrolmentForm.tsx

```diff
@@ -232,10 +234,14 @@ const EnrolmentForm: React.FC<Props> = ({
               <div className={styles.divider} />
 
               <h2>{t('enrolment:enrolmentForm.titleNotifications')}</h2>
+              {errors.hasEmailNotification && touched.hasEmailNotification && (
+                <ErrorMessage>{t(errors.hasEmailNotification)}</ErrorMessage>
+              )}
```

Comment:
email notifications are always checked and disabled, why is this needed?

Reply:
Maybe it is not. I didn't think about it much. I thought there could be a way to achieve it some how...

I will take it off.